sword2/__init__.py

```diff
@@ -1,17 +1,17 @@
 """
 SWORD2 Python Client blurb
 """
-from service_document import ServiceDocument
-from collection import SDCollection, Collection_Feed
-from statement import Atom_Sword_Statement, Ore_Sword_Statement
-from error_document import Error_Document
-from connection import Connection
-from transaction_history import Transaction_History
-from exceptions import *
-from server_errors import SWORD2ERRORSBYIRI, SWORD2ERRORSBYNAME
-from utils import Timer, NS, get_md5, create_multipart_related
-from implementation_info import *
-from atom_objects import Entry, Category
-from http_layer import HttpLayer, HttpResponse, HttpLib2Layer, UrlLib2Layer
-from auto_discovery import AutoDiscovery
-from deposit_receipt import Deposit_Receipt
+from .service_document import ServiceDocument
```

Comment:
Can you replace relative imports with absolute ones? It's not necessary, but relative imports obfuscates the code readability :/